apps/minaintyg/src/pages/Certificate/components/CertificateFooter.tsx

```diff
@@ -3,7 +3,12 @@ import { CertificateMetadata } from '../../../schema/certificate.schema'
 
 export function CertificateFooter({ issuer, unit, careUnit }: CertificateMetadata) {
   const unitName = unit.name === careUnit.name ? unit.name : [unit.name, careUnit.name].join(', ')
-  const unitAddress = [unit.address, unit.zipCode, unit.city].join(', ')
+
+  function isUndefinedOrEmpty(field: string) {
+    return field && field.length > 0
+  }
+
+  const unitAddress = [unit.address, unit.zipCode, unit.city].filter((field) => isUndefinedOrEmpty(field)).join(', ')
```

Comment:
`.filter((field) => isUndefinedOrEmpty(field))` === `.filter(isUndefinedOrEmpty)`

I wonder if `isTruthy` could be used here instead ?

Reply:
Ill check it out!